cmd/fyne/internal/commands/package.go

```diff
@@ -523,10 +523,9 @@ func validateAppID(appID, os, name string, release bool) (string, error) {
 			// appID package names can not start with '_' or a number
 			packageNames := strings.Split(appID, ".")
 			for _, name := range packageNames {
-				r := []rune(name)
-				if r[0] == '_' {
+				if name[0] == '_' {
```

Comment:
Potential safety issue here - if the string contains ".." won't this panic?

Reply:
Good point. I will fix this